src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -189,28 +191,47 @@ function epidemicModelReducer(
 
       // change in state or county triggers a bigger reset
       let updates = { ...action.payload };
+
       let { stateCode, countyName, countyLevelData } = updates;
 
       countyLevelData = countyLevelData || state.countyLevelData;
+
+      let newState;
```

Comment:
Nit:  I think I might be confusing myself a bit by conflating what "state" is meant to represent in the variables such as `newState` and `stateInitialized`.  In those cases, is "state" meant to represent a political territory (i.e. "Virginia", "California, etc.) or is "state" meant to represent the current status of the data?

Reply:
It's in the latter sense, of the model data state. Definitely confusing juxtaposition.